pkg/kubelet/clustertrustbundle/clustertrustbundle_manager.go

```diff
@@ -40,6 +46,42 @@ const (
 	maxLabelSelectorLength = 100 * 1024
 )
 
+// clusterTrustBundle is a type constraint for version-independent ClusterTrustBundle API
+type clusterTrustBundle interface {
+	certificatesv1alpha1.ClusterTrustBundle | certificatesv1beta1.ClusterTrustBundle
+}
+
+// clusterTrustBundlesLister is an API-verion independent ClusterTrustBundles lister
+type clusterTrustBundlesLister[T clusterTrustBundle] interface {
+	Get(string) (*T, error)
+	List(labels.Selector) ([]*T, error)
+}
+
+type clusterTrustBundleHandlers[T clusterTrustBundle] interface {
+	GetSignerName(*T) string
+	GetTrustBundle(*T) string
+}
+
+type alphaClusterTrustBundleHandlers struct{}
```

Comment:
Same question about handling alpha APIs here... isn't it fine to just straight-up break people who were using the alpha API?